pkg/network/server.go

```diff
@@ -968,6 +969,38 @@ func (s *Server) broadcastTxHashes(hs []util.Uint256) {
 	})
 }
 
+// broadcastStaleTxLoop is a loop for rebroadcasting tx which are in a mempool
+// for too long.
+func (s *Server) broadcastStaleTxLoop() {
```

Comment:
And I don't think we need this. We only need `Server` to give `s.transactions` channel to the mempool (maybe along with `resendThreshold`).

Reply:
Fixed.